types/setting.go

```diff
@@ -462,6 +465,16 @@ var (
 		Default:     "25",
 	}
 
+	SettingDefinitionStorageReservedPercentage = SettingDefinition{
+		DisplayName: "Storage Reserved Percentage",
```

Comment:
As I said, change the name accordingly, so please review the whole PR..

Reply:
Updated all. Thanks. Only func `CreateDefaultDisk` keeps the parameter name `storageReservedPercentage`